learning/features/3d/3d_performance_and_limitations.rst

```diff
@@ -27,10 +27,10 @@ usability and flexibility. Some practical examples of this are:
 
 And there are many more examples of this! Game engines strive to be
 general purpose in nature, so balanced algorithms are always favored
-over algorithms that might be the fast in some situations and slow in
+over algorithms that might be the fastest in some situations and slow in
```

Comment:
Scans better if you leave it "fast" but remove "the" -> "fast in some ... and slow in others"